drawee/src/main/java/com/facebook/drawee/drawable/RoundedBitmapDrawable.java

```diff
@@ -283,6 +286,10 @@ private void updateTransform() {
 
   private void updatePath() {
     if (mIsPathDirty) {
+      mBitmapBounds.set(0, 0, getBitmap().getWidth(), getBitmap().getHeight());
+      mTransform.mapRect(mBitmapBounds);
+      //mRootBounds.intersect(mBitmapBounds);
```

Comment:
This line should be removed. Uncommenting it would make the rounded path be like on the picture, but this is actually not desirable because of the border. Namely, there might be other branches such as background or placeholder and all of them should have the same rounded border.
 
![screen shot 2016-06-08 at 20 02 40](https://cloud.githubusercontent.com/assets/3705123/15941175/d372317e-2e76-11e6-895f-40f1d8d1d5fe.png)